workers/generic-worker/graceful/graceful.go

```diff
@@ -0,0 +1,30 @@
+package graceful
```

Comment:
Let's add a package comment here to say this package isn't thread safe, and isn't intended to be. This can act as a warning for us if we start using it for other things. But since we only have a single calling go routine for now, this shouldn't be a problem.